src/libs/actions/Wallet.js

```diff
@@ -321,6 +321,31 @@ function answerQuestionsForWallet(answers, idNumber) {
     );
 }
 
+function requestPhysicalExpensifyCard(cardID, updatedPersonalDetails) {
+    // TODO: confirm required parameters
```

Comment:
@grgia 

Can you tell me what parameters are missing here and confirm if the onyx data that is being sent it the correct one? Thanks